index.html

```diff
@@ -1427,6 +1323,149 @@ <h4>EcdsaSecp256k1RecoveryMethod2020</h4>
+        <h4>LinkedDomains</h4>
+        <div class="issue" data-number="167"></div>
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Normative Definition</th>
+              <th>JSON-LD</th>
+              <th>CDDL</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <a href="https://identity.foundation/.well-known/resources/did-configuration/">Well Known DID Configuration</a>
+              </td>
+              <td>
+                <a href="https://identity.foundation/.well-known/did-configuration/v1">Well Known DID Configuration</a>
+              </td>
+              <td>
+                <span>CDDL definition pending</span>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <pre class="example" title="Example of service and serviceEndpoint properties">
+{
+  "@context": ["https://www.w3.org/ns/did/v1","https://identity.foundation/.well-known/did-configuration/v1"],
+  "id": "did:example:123",
+  "verificationMethod": [{
+    "id": "did:example:123#456",
+    "type": "JsonWebKey2020",
+    "controller": "did:example:123",
+    "publicKeyJwk": {
+      "kty": "OKP",
+      "crv": "Ed25519",
+      "x": "VCpo2LMLhn6iWku8MKvSLg2ZAoC-nlOyPVQaO3FxVeQ"
+    }
+  }],
+  "service": [
+    {
+      "id":"did:example:123#foo",
+      "type": "LinkedDomains",
+      "serviceEndpoint": {
+        "origins": ["https://foo.example.com", "https://identity.foundation"]
+      }
+    },
+    {
+      "id":"did:example:123#bar",
+      "type": "LinkedDomains",
+      "serviceEndpoint": "https://bar.example.com"
+    }
+  ]
+}
+        </pre>
+
+      </section>
+
+      </section>
+
+  </section>
+
+  <section>
+    <h1>Representation-Specific Entries</h1>
+
+    <section>
+      <h3>JSON</h3>
+
+      <p>
+These are entries in DID documents that are specific to the
+<a href="https://w3c.github.io/did-core/#json">JSON representation</a>.
+      </p>
+      <p>
+(No entries yet)
+      </p>
+    </section>
+
+    <section>
+      <h3>JSON-LD</h3>
+
+      <p>
+These are entries in DID documents that are specific to the
+<a href="https://w3c.github.io/did-core/#json-ld">JSON-LD representation</a>.
+      </p>
+
+      <section>
+        <h4>@context</h4>
+
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Normative Definition</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <a href="https://www.w3.org/TR/did-core/#production-0">DID Core</a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p class="issue">
+          This is an active area of debate regarding if <code>@context</code> belongs as a base property in the 
+          DID Core. See <a href="https://github.com/w3c/did-core/issues/432">issue
+          #283</a>.
```

Comment:
```suggestion
There is active debate regarding whether <code>@context</code>
belongs as a base property in the DID Core. See
<a href="https://github.com/w3c/did-core/issues/432">issue #283</a>.
```

Reply:
Actually since that issue in DID Core is already closed, I think we should just delete this whole paragraph

Reply:
Irksome... I find no way to *cancel* this suggestion (which left out the `<p>...</p>` tags, since it was correcting stuff within them) in favor of the newer suggestion below.

Reply:
You can resolve the thread in github with a comment... or I can apparently delete your suggestion....

Reply:
I have no "resolve" option. I can comment, or I can delete (which loses the comment from @peacekeeper which led to my subsequent larger edit)... Github is weird.